openedx/core/djangoapps/programs/signals.py

```diff
@@ -47,5 +47,5 @@ class of the object instance that sent this signal
         status,
     )
     # import here, because signal is registered at startup, but items in tasks are not yet able to be loaded
-    from openedx.core.djangoapps.programs import tasks
-    tasks.award_program_certificates.delay(user.username)
+    from openedx.core.djangoapps.programs.tasks.v1.tasks import award_program_certificates
```

Comment:
i'm not feeling it.  If we add tasks related to other programs use cases, this would mean a second separate directory tree with identical structure.  Also, I don't want to muddy the waters around how/why we are versioning subpackages; `tasks.vN` keeps that pretty clear.